Makefile

```diff
@@ -91,7 +91,7 @@ build: $(BIN_NAME)
 
 .PHONY: deps
 deps: go.mod go.sum
-	go mod tidy
+	#go mod tidy
```

Comment:
This seems wrong
```suggestion
	go mod tidy
```

Reply:
Well, I know. But thats whats reverts my dependency update: https://github.com/observatorium/api/pull/604#issuecomment-1825350148